### DISCUSSION TITLE
How should I add pages ?

### DISCUSSION THREAD

**Discussion by zerbusdetroy** [skoropadas/ng-doc#135] [Q&A] [answered]
Hello,
I must be missing something but I don't understand how ng-doc works.

Here are the steps I've followed:

- Created a new Angular application (version 17.2), with and without SSR.
- Installed ng-doc using the command `ng add @ng-doc/add`.
- Generated a new page using the command `ng g @ng-doc/builder:page "Installation"`.
- Started the application using `ng start`.

Despite these steps, the documentation is empty.
![Capture d’écran 2024-02-28 à 17 14 38](https://github.com/ng-doc/ng-doc/assets/17167593/03f556f8-8c28-45cd-8598-0df650fbabb5)

I'm unsure what I might be missing or if there's an issue with ng-doc. Any guidance or assistance would be greatly appreciated.

Thank you !

**Comment by skoropadas** [comment, marked as answer]
Hi, by default, NgDoc uses your project's sourceRoot as the directory where you should create documentation, so check pls that you created your page in your sourceRoot folder, usually it's `src` folder of your app
